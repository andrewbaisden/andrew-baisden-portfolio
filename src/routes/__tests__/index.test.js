import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import index from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('<index />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<index />);
    expect(wrapper).toMatchSnapshot();
  });
});
