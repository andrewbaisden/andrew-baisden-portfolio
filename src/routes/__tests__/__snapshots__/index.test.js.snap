// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<index /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
